### ISSUE TITLE
Can Mujoco's camera produce a depth of field effect?

### ISSUE THREAD

**Issue by LinShuang22** [google-deepmind/mujoco#1526]
Hi,

I'm a student and I'm trying to use MuJoCo for simulating a camera as realistic as possible.

I'm looking for some help  to simulate depth of field for the camera in Mujoco. I need to determine if an object has reached a narrow range by judging the clarity of the image feedback from the camera.

I want to achieve the capability for the camera to only see objects clearly within a relatively small area. Moving away from this area (forward or backward) will result in the camera seeing a blurry image. 

I noticed the issue #1114 , and I tried model https://github.com/google-deepmind/mujoco/blob/main/test/engine/testdata/vis_visualize/frustum.xml . I tried changing `znear`, `zfar`, `frustum`, but even though the object moved away from the frustum (behind it), I could still see the object clearly without it becoming blurry. Did I modify the parameters incorrectly? I also tried changing `focal` and other parameters. When the camera approaches a certain distance, the image returned by the camera becomes blurred, but it doesn't blur when moving away.

Which parameters should I modify to ensure that the camera can only see clear images within a certain range of distances? Also, are parameters like `znear` global? Can I customize `znear` for individual cameras? Could you provide me with an example model?

![图片](https://github.com/google-deepmind/mujoco/assets/119031680/bcf88d66-9426-44d9-8305-91ee43b728fb)

blurry image like this:
![图片](https://github.com/google-deepmind/mujoco/assets/119031680/a7878286-49e5-4e4b-89a8-0895abf27a3a)

**Comment by yuvaltassa** [comment]
No. Currently MuJoCo's native cameras are point cameras and only render perfect focus everywhere. The parameters you mentioned are used for real camera calibration, but depth of field is not rendered. We are in the process of allowing export of MuJoCo visuals to USD format, which would let you use more advanced renderers which have these effects. Look at the `python/mujoco/usd` directory, though note this is work in progress. cc @awesome-aj0123
